src/components/App/test/wdio/default.js

```diff
@@ -0,0 +1,14 @@
+const { assertScreenMatches } = require('../../../../utils/webdriverutils');
+
+describe('webdriver.io page', () => {
+    it('should have the right title', () => {
+        browser.url('/iframe.html?id=app--default');
+
+        browser.execute(() => {
+            document.querySelector('.App-logo').style.animation= "none";
+        });
+
+        var screenshots = browser.checkElement('.App');
+        assertScreenMatches(screenshots);
+    });
+});
```

Comment:
See generator comments...

Reply:
Ran the linter. I think this is good now.